### DISCUSSION TITLE
Scrollpsy

### DISCUSSION THREAD

**Discussion by mvi3** [bootscore/bootscore#478] [Q&A] [answered]
Hello,

I can't change scrollpsy for top menu, i wanna scroll section to maximum top site, now website is scrolling to botton menu but menu is transparent and i have breake in top, what i need change ? I adding jquery for template with active manu this is working fine.
```
jQuery(function ($) {

  // Activate Scrollspy
  $('.entry-content').scrollspy({
    target: '#nav-main'
  });

  // Refresh Scrollspy
  $('[data-spy="scroll"]').each(function () {
    $(this).scrollspy("refresh");
  });

});
```
(https://github.com/bootscore/bootscore/assets/134155757/e42450d4-45f6-42c5-8ffc-78442608e817)

**Comment by crftwrk** [comment, marked as answer]
The only thing I see is that you should use `#bootscore-navbar` instead `#nav-main`. But your issue is impossible to check without having a link to your site. Please share always a link to your project https://bootscore.me/documentation/support-contribute/#What_we_do.

Thanks ;-)
